lib/order.rb

```diff
@@ -0,0 +1,63 @@
+require 'twilio-ruby'
+
+class Order
+
+  attr_reader :selection
+  attr_reader :menu
+
+  def initialize menu
+    @selection = {}
+    @menu = menu
+  end
+
+  def add_item dish, quantity
+    if !on_menu? dish
+      return "That item is not on the menu"
+    else
+      selection.store dish, quantity
+    end
```

Comment:
I had it like your example to begin with, but changed it to the former for readability. What do you think?